.github/workflows/publish_docs.yml

```diff
@@ -13,7 +13,7 @@ permissions:
 jobs:
   build:
     name: "Build Docs"
-    runs-on: 'ubuntu-latest'
```

Comment:
I imagine we'll get questions about why latest is being changed - wouldn't that not use 20.04? I'm assuming it's because it's better to have pinned versions but it still confused me.